enrolment-ui/src/modules/Applications/redux.js

```diff
@@ -109,7 +109,7 @@ export const reducer = (state = _defaultState, action) => {
       return {
         ...state,
         loading: false,
-        hasApplications: action.payload.count > 0,
+        hasApplications: action.payload.length > 0,
```

Comment:
Dependency changes in the API remove counts from the response, so now we get the count by the length of the response array.